stdlib/src/python/_cpython.mojo

```diff
@@ -199,19 +199,17 @@ struct PyObjectPtr:
         # TODO: Make this part of the trait bound
         expected_type_name: StringLiteral,
     ) -> Optional[UnsafePointer[T]]:
-        var cpython = _get_global_python_itf().cpython()
-
-        var type = cpython.Py_TYPE(self)
-
-        var type_name = PythonObject(cpython.PyType_GetName(type))
+        cpython = _get_global_python_itf().cpython()
```

Comment:
I asked Joe, and it seems that the in-house style is still using `var`s.

Reply:
I've already gotten several PRs with no `var` approved :man_shrugging:, and I've seen it written in `_cpython.mojo` as well. IMO resistance is futile in this case if we want to go in the Python direction (several PRs will eventually leak variables with no var and we'll lose consistency). `var` is currently only really necessary when declaring a variable's type or when using variables with the same name and different types in `if` branches. Though if somebody from the team tells me to, I'll change them np.

Reply:
> I've already gotten several PRs with no var approved

Me too. Maybe let's summon @JoeLoser here for a definite answer.

Reply:
Does my comment at https://github.com/modularml/mojo/pull/3705#discussion_r1824760343 help clarify things? I haven't blocked PRs in the past on it until we had a chance to talk as a team.

Reply:
@JoeLoser  FYI I already reverted the var removal in this PR. Some others we can just leave for a cleanup PR, I think it will be easier to just go over each file 1 time after merging a bunch :man_shrugging:.